CRM/Core/Form.php

```diff
@@ -387,6 +387,14 @@ public function &add(
     }
     // @see http://wiki.civicrm.org/confluence/display/CRMDOC/crmDatepicker
     if ($type == 'datepicker') {
+      // Support minDate/maxDate properties
+      if (isset($extra['minDate'])) {
+        $extra['minDate'] = date('Y-m-d', strtotime($extra['minDate']));
```

Comment:
sigh it should really already be in Y-m-d format :-(

Reply:
Oh I see - this also supports + 15 days - that makes sense then - I thought it was just us doing weird format handling